src/overmap.cpp

```diff
@@ -946,6 +951,39 @@ void oter_type_t::load( const JsonObject &jo, const std::string_view )
     optional( jo, was_loaded, "connect_group", connect_group, string_reader{} );
     optional( jo, was_loaded, "travel_cost_type", travel_cost_type, oter_travel_cost_type::other );
 
+    if( jo.has_string( "default_map_data" ) ) {
+        optional( jo, was_loaded, "default_map_data", default_map_data );
+    } else {
```

Comment:
This prevents copy-from from working with this, because it will always be overwritten.
```suggestion
    } else if( !was_loaded ) {
```

Reply:
To clarify, if it's not a copy-from entry, was_loaded will be false at this point and apply the fallback, if it IS a copy-from entry it will be true and only overwrite if there is a "default_map_data" field?
(I'm seeing a seemingly related crash after applying this)

Reply:
Yes. To ensure that the optional isn't default initializing default_map_data (for some reason), consider replacing it with `mandatory` (because it is when the member is defined).

example:
B copies-from A.

Loading A creates a new entity, so `A.was_loaded` is false.
A defines `"default_map_data"` `foo`.
after loading, `A.was_loaded` is now true.

Loading B creates an entity that is a copy of A, so `B.was_loaded` is true.
If B does not define `"default_map_data"`, this else case is hit, and it does not keep `foo`, as it should.